jdk_other: javax/xml/ws/8172297/Main.java ACTION: testng -- Failed. Execution failed: `main' threw exception: java.lang.Exception: failures: 1

23:55:21 ACTION: testng -- Failed. Execution failed: `main' threw exception: java.lang.Exception: failures: 1
23:55:21 REASON: User specified action: run testng/othervm Main 
23:55:21 TIME:   4.446 seconds
23:55:21 messages:
23:55:21 command: testng Main
23:55:21 reason: User specified action: run testng/othervm Main 
23:55:21 Mode: othervm [/othervm specified]
23:55:21 elapsed time (seconds): 4.446
23:55:21 configuration:
23:55:21 STDOUT:
23:55:21 [TestNG] Running:
23:55:21   javax/xml/ws/8172297/Main.java
23:55:21 
23:55:21 Publishing WebService on 41603 port
23:55:21 Waiting for done signal from test client.
23:55:21 Launching WS client connection on 41603 port
23:55:21 Client input parameter: '#CR#CR#NL#CR#CR CarriageReturn and NewLine #CR#NL#NL Test #CR#CR#CR#CR'
23:55:21 Server-side parameter value: '#NL#NL#NL#NL CarriageReturn and NewLine #NL#NL Test #NL#NL#NL#NL'
23:55:21 Client response parameter: '#NL#NL#NL#NL CarriageReturn and NewLine #NL#NL Test #NL#NL#NL#NL'
23:55:21 Got done signal from the client. Stopping WS endpoint.
23:55:21 test Main.runTest(): failure
23:55:21 java.lang.AssertionError: WS parameter was modified during round trip. expected [false] but found [true]
23:55:21 	at org.testng.Assert.fail(Assert.java:94)
23:55:21 	at org.testng.Assert.failNotEquals(Assert.java:496)
23:55:21 	at org.testng.Assert.assertFalse(Assert.java:63)
23:55:21 	at Main.runTest(Main.java:65)
23:55:21 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
23:55:21 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
23:55:21 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


Tests:
javax/xml/ws/8172297/Main.java
javax/xml/bind/xsom/8159240/WhitespacesTest.java

--- a/buildenv/jenkins/Jenkinsfile_personal
+++ b/buildenv/jenkins/Jenkinsfile_personal
@@ -8,7 +8,10 @@ pipeline {
 		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
 		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
 		string (defaultValue: '', description: 'Specific test project, leave blank by default for all projects to be compiled, e.g. openjdk_regression systemtest performance jck thirdparty_containers', name: 'TESTPROJECT')
-		string (defaultValue: 'runtest', description: 'Test Target, default is all target(runtest), e.g.openjdk sanity extended test_systemtest test_DirectoryName, jdk_bean_SE80_0, etc', name: 'TARGET')
+		string (defaultValue: 'runtest', description: 'Test Target to execute, default is runtest which will run all tests. You can also select sub-targets such as:
+\ntop level: openjdk, sanity, extended, perf, etc
+\ndirectory level: prefix a directory with test (eg test_systemtest)
+\nindividual test: jdk_bean_SE80_0, etc', name: 'TARGET')
 		choice (choices: 'linux&&x64&&test\nmac&&x64&&test\nwindows&&x64&&test', description: 'What is machine label?', name: 'MACHINE_LABEL')
 		choice (choices: 'releases\nnightly\nupstream\ncustomized', description: 'Where is sdk?', name: 'SDK_RESOURCE')
 		string (defaultValue: "N/A", description: 'Upstream build name, only set when SDK_RESOURCE is upstream', name: 'UPSTREAM_JOB_NAME')

--- a/openjdk_regression/ProblemList_openjdk8.txt
+++ b/openjdk_regression/ProblemList_openjdk8.txt
@@ -84,11 +84,6 @@ gc/g1/TestStringDeduplicationYoungGC.java	116	generic-all
 gc/g1/TestStringSymbolTableStats.java	116	generic-all
 gc/logging/TestGCId.java	116	generic-all
 gc/whitebox/TestWBGC.java	116	generic-all
-runtime/ClassUnload/KeepAliveClass.java		120	generic-all
-runtime/ClassUnload/KeepAliveClassLoader.java	120	generic-all
-runtime/ClassUnload/KeepAliveObject.java	120	generic-all
-runtime/ClassUnload/KeepAliveSoftReference.java	120	generic-all
-runtime/ClassUnload/UnloadTest.java	120	generic-all
 runtime/Metaspace/FragmentMetaspaceSimple.java 120	generic-all
 runtime/ErrorHandling/TestCrashOnOutOfMemoryError.java	121	generic-all
 runtime/ErrorHandling/TestExitOnOutOfMemoryError.java	121	generic-all
@@ -164,10 +159,6 @@ java/lang/management/MemoryMXBean/LowMemoryTest2.sh	157	linux-all
 ############################################################################
 
 # jdk_math
-
-############################################################################
-
-# jdk_other
 
 ############################################################################
 
@@ -350,6 +341,5 @@ java/util/ResourceBundle/Control/Bug6530694.java	137	macosx-all
 ############################################################################
 
 # jdk_other
-javax/xml/ws/8172297/Main.java	119 generic-all
-javax/xml/bind/xsom/8159240/WhitespacesTest.java	119 generic-all
+
 ############################################################################

